unblob/handlers/executable/elf.py

```diff
@@ -18,6 +19,49 @@
 
 lief.logging.set_level(lief.logging.LOGGING_LEVEL.ERROR)
 
+logger = get_logger()
+
+# length is encoded in at most 2 bytes starting after the 2nd byte - 30 82 ?? ??
+ASN1_PREFIX_LEN = 4
+ASN1_SEQUENCE_START = b"\x30"
+KERNEL_MODULE_SIGNATURE_PADDING_LEN = 12
```

Comment:
this would be actually the signature info :+1: 
```
struct module_signature {
        uint8_t         algo;           /* Public-key crypto algorithm [0] */
        uint8_t         hash;           /* Digest algorithm [0] */
        uint8_t         id_type;        /* Key identifier type [PKEY_ID_PKCS7] */
        uint8_t         signer_len;     /* Length of signer's name [0] */
        uint8_t         key_id_len;     /* Length of key identifier [0] */
        uint8_t         __pad[3];
        uint32_t        sig_len;        /* Length of signature data */
};
```

I would maybe call this `KERNEL_MODULE_SIGNATURE_INFO_LEN`

Reply:
Added `module_signature` parsing in the latest revision.